src/sections/Planning/PlanningCard.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import "./PlanningCard.scss";
+import PlanningCardHover from './PlanningCardHover';
+
+const PlanningCard = ({image,title,description, children}) => {
+  const [style, setStyle] = React.useState({display: 'block'});
+  const [styleHover, setStyleHover] = React.useState({display: 'none'});
+  return (
+    <>
+      <div className='planning-card' style={style} onMouseEnter={e => {
+        setStyle({display: 'none'});
+        setStyleHover({display: 'block'});
+      }}>
+        <img src={image}/>
+        <h3 >{title}</h3>
+        <p>{description}</p>
+      </div>
+      <PlanningCardHover className="hover-content" style={styleHover} setStyle={setStyle} setStyleHover={setStyleHover}>
```

Comment:
why we call it PlanningCardHover when this is not always going to be in hover state? hover is a state, do we want to name component in reference to a state name?

Reply:
We are showing this component only on hover(mouse enter into parent)